packages/clay-autocomplete/src/Autocomplete.tsx

```diff
@@ -404,13 +416,50 @@ function AutocompleteInner<T extends Item>(
 		),
 		items: filteredItems,
 		notFound: (
-			<DropDown.Item
-				aria-disabled="true"
-				className="disabled"
-				roleItem="option"
-			>
-				{messages.notFound}
-			</DropDown.Item>
+			<>
+				{allowsCustomValue && (
+					<>
+						<DropDown.Item
+							className=""
+							id={value}
+							key={value}
+							onClick={() => {
+								if (allowsCustomValue && items && onAddNewItem) {
+									onAddNewItem();
+
+									items.push(value);
+
+									inputElementRef.current?.focus();
+
+									setValue('');
+								}
+							}}
+							roleItem="option"
+						>
+							{messages.setAsHTML &&
```

Comment:
We don't need to allow setting html. The designs don't provide this flexibility so we shouldn't either..

Reply:
@ethib137 in some of the documentation there were some differences that required flexibility.

[Case 1](https://www.figma.com/design/8iouiTNmXq56OcuhWQEGry/LPD-51807-STOR-Autocomplete---Dropdown-with-fixed-button?node-id=1-476&p=f&t=VIO5MLaTjCeTP6FY-0): Has parts of the message italicized and message text after the value

![case-1](https://github.com/user-attachments/assets/98ce0056-88a2-492a-ae53-fa52ccefcc5c)

[Case 2](https://liferay.atlassian.net/wiki/spaces/PRDDXP/pages/3811442825/Create+New+Option#%F0%9F%8E%A8--Design): No italics and message is before the value

![case-2](https://github.com/user-attachments/assets/1fb864a2-8937-4181-9712-6ccbd683700a)

HTML passed in the message prop gets escaped. I believe the only way to support this is through setting HTML.